### PR TITLE
Issue #93 Hispanic Origin Question 

### PR DESCRIPTION
**Pull request by crystallynnvoss**
Ticket Description
Fixes issue #93 

Description of Changes
Update survey to include additional options for hispanic origin question 

Before and After for UI Updates
<-- You should have 2 images minimum, for desktop, 4 for mobile and desktop combined -->

Before:

After:

For PR Reviewer
 Does this file change the yarn.lock, package.json or package-lock.json file? If so, why?
 If this pr contains mobile and desktop changes, did you test on IphoneXr and desktop views?
 Does this file match the related tickets linked figma file, or does it pass the visual smell test?
 If this file contains javascript, does the javascript pass the smell test?
 If you don't feel super confident in your review, did you assign someone more senior to double check?

### CODE REVIEW

File: src/components/pages/SurveyPage.jsx [hunk 1]
```diff
@@ -367,7 +375,7 @@ export const SurveyPageForm = withFormik({
 })(SurveyForm);
 
 function SurveyPage() {
-  const auth = useAuth();
+  // const auth = useAuth();
```
Comment (YolineB) [inline src/components/pages/SurveyPage.jsx:378]: This can be uncommented

File: src/components/pages/SurveyPage.jsx [hunk 2]
```diff
@@ -381,8 +389,8 @@ function SurveyPage() {
   };
 
   // TODO: Move into useAuth
-  if (!auth.user) return <div>Loading</div>
-  if (!auth.user.isAuthenticated) return <div>Unauthenticated</div>
+  // if (!auth.user) return <div>Loading</div>
+  // if (!auth.user.isAuthenticated) return <div>Unauthenticated</div>
```
Comment (YolineB) [inline src/components/pages/SurveyPage.jsx:393]: These can be uncommented too :)

File: src/App.js [hunk 3]
```diff
@@ -46,11 +46,11 @@ function App() {
             <Route
               path="/surveys/:surveyId"
               element={
-                <RequireAuth>
+                // <RequireAuth>
                   <SurveyPage />
-                </RequireAuth>
+                // </RequireAuth>
```
Comment (YolineB) [inline src/App.js:51]: These can also be uncommented

**Review by YolineB — CHANGES_REQUESTED** [review]
Just the uncommenting and then it is good!

**Comment by crystallynnvoss** [comment]
@YolineB ok should be good to go

**Review by YolineB — APPROVED** [review]
LGTM!

**Pull request merged**